modules/installation-osp-default-kuryr-deployment.adoc

```diff
@@ -0,0 +1,313 @@
+https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/13/html-single/networking_guide/#planning_your_octavia_deployment[installation
+of Octavia using Director].
+====
+
+[NOTE]
+====
+If Kuryr SDN is used, the Overcloud installation requires the `trunk` extension
+to be enabled at Neutron. This is enabled by default on Director deployments.
+Use the `openvswitch` firewall instead of the default `ovs-hybrid` when the Neutron
+backend is ML2/OVS. There is no need for modifications if the backend is
+ML2/OVN.
+====
+
+To enforce Network Policies across Services, like when traffic goes through
+the Octavia load balancer, we need to ensure Octavia creates the amphora VM
+security groups on the user project. To do that, you must add the project ID
+to the `octavia.conf` configuration file after you create the project.
+This step ensures that required LoadBalancer security groups belong to that
+project and that they can be updated to enforce Services isolation.
+
+. Get the project ID
++
+----
+$ openstack project show *<project>*
++-------------+----------------------------------+
+| Field       | Value                            |
++-------------+----------------------------------+
+| description |                                  |
+| domain_id   | default                          |
+| enabled     | True                             |
+| id          | PROJECT_ID                       |
+| is_domain   | False                            |
+| name        | *<project>*                      |
+| parent_id   | default                          |
+| tags        | []                               |
++-------------+----------------------------------+
+----
++
+. Add the project ID to `octavia.conf` for the controllers and restart the
+Octavia worker.
++
+----
+$ source stackrc  # undercloud credentials
+$ openstack server list
++--------------------------------------+--------------+--------+-----------------------+----------------+------------+
+│
+| ID                                   | Name         | Status | Networks
+| Image          | Flavor     |
+│
++--------------------------------------+--------------+--------+-----------------------+----------------+------------+
+│
+| 6bef8e73-2ba5-4860-a0b1-3937f8ca7e01 | controller-0 | ACTIVE |
+ctlplane=192.168.24.8 | overcloud-full | controller |
+│
+| dda3173a-ab26-47f8-a2dc-8473b4a67ab9 | compute-0    | ACTIVE |
+ctlplane=192.168.24.6 | overcloud-full | compute    |
+│
++--------------------------------------+--------------+--------+-----------------------+----------------+------------+
+
+$ ssh heat-admin@192.168.24.8  # ssh into the controller(s)
+
+controller-0$ vi /var/lib/config-data/puppet-generated/octavia/etc/octavia/octavia.conf
+[controller_worker]
+# List of project ids that are allowed to have Load balancer security groups
+# belonging to them.
+amp_secgroup_allowed_projects = PROJECT_ID
+
+controller-0$ sudo docker restart octavia_worker
+----
```

Comment:
There's a lot going on in this step. It's hard to digest all this information at once. What do you think about breaking it up into multiple steps? (list OpenStack servers, ssh into controller, restart Octavia, etc.).

Reply:
Let me add comments there...